src/telemetry/writer.cpp

```diff
@@ -22,10 +22,24 @@
 
 #include "writer.hpp"
 #include "data/data.hpp"
+#include "chrono"
+#include "cstdint"
+#include "iostream"
 
 namespace hyped {
 namespace telemetry {
 
+
+
+// The current time in milliseconds that will be used later
+void Writer::packTime()
+{
+  rjwriter_.Key("time");
+  using namespace std::chrono;
```

Comment:
Would be better to just prepend the necessary keywords with `std::chrono` instead of including the namespace

Reply:
I am sorry but I don't understand what do you mean, can you explain more?

Reply:
It would be better to remove the line `using namespace std::chrono;`

and in the line below to use
`std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count()`

Reply:
Thanks, I have adapted the change and pushed the new codes